src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Arm/AdvSimd.PlatformNotSupported.cs

```diff
@@ -46,6 +46,66 @@ public new abstract class Arm64 : ArmBase.Arm64
             /// </summary>
             public static Vector128<double> Add(Vector128<double> left, Vector128<double> right) { throw new PlatformNotSupportedException(); }
 
+            /// <summary>
+            /// uint8_t vaddv_u8(uint8x8_t a)
+            ///   A64: ADDV Bd, Vn.8B
+            /// </summary>
+            public static byte AddAcross(Vector64<byte> value) { throw new PlatformNotSupportedException(); }
+
+            /// <summary>
+            /// int16_t vaddv_s16(int16x4_t a)
+            ///   A64: ADDV Hd, Vn.4H
+            /// </summary>
+            public static short AddAcross(Vector64<short> value) { throw new PlatformNotSupportedException(); }
+
+            /// <summary>
+            /// int8_t vaddv_s8(int8x8_t a)
+            ///   A64: ADDV Bd, Vn.8B
+            /// </summary>
+            public static sbyte AddAcross(Vector64<sbyte> value) { throw new PlatformNotSupportedException(); }
+
+            /// <summary>
+            /// uint16_t vaddv_u16(uint16x4_t a)
+            ///   A64: ADDV Hd, Vn.4H
+            /// </summary>
+            public static ushort AddAcross(Vector64<ushort> value) { throw new PlatformNotSupportedException(); }
+
+            /// <summary>
+            /// uint8_t vaddvq_u8(uint8x16_t a)
+            ///   A64: ADDV Bd, Vn.16B
+            /// </summary>
+            public static byte AddAcross(Vector128<byte> value) { throw new PlatformNotSupportedException(); }
+
+            /// <summary>
+            /// int16_t vaddvq_s16(int16x8_t a)
+            ///   A64: ADDV Hd, Vn.8H
+            /// </summary>
+            public static short AddAcross(Vector128<short> value) { throw new PlatformNotSupportedException(); }
+
+            /// <summary>
+            /// int32_t vaddvq_s32(int32x4_t a)
+            ///   A64: ADDV Sd, Vn.4S
+            /// </summary>
+            public static int AddAcross(Vector128<int> value) { throw new PlatformNotSupportedException(); }
```

Comment:
There is no `Vector64<int>` overload because it is equivalent to a `AddPairwise`, correct?

Reply:
(so the manual marks `size = 10, Q = 0` and `size = 11, Q = 1` as reserved; with `size = 11, Q = 0` as reserved because it would be a nop).

Reply:
Yeah, `vector64<int>` would be an `ADDP`, but I couldn't see anyway to encode that in the intrinsics table in `hwintrinsiclistarm64.h` as it assumes you have the same instruction for all sizes of the same type unless I'm mistaken? It doesn't seem to allow overloading on the maximum vector size if the name stays the same.

Reply:
> It doesn't seem to allow overloading on the maximum vector size if the name stays the same.

Right, you would need to have special-handling in codegen to ensure the right instruction was selected.

Given that the underlying instruction encoding on `ADDV` is blocked and users have the same functionality via the explicit `AddPairwise`, I think not exposing it is the right decision.